GitUI/Native.cs

```diff
@@ -34,9 +34,45 @@ internal struct FORMATRANGE
             public CHARRANGE chrg;         //Range of text to draw (see earlier declaration)
         }
 
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
+        internal struct LOGFONT
+        {
+            public const int LF_FACESIZE = 32;
+            public int lfHeight;
+            public int lfWidth;
+            public int lfEscapement;
+            public int lfOrientation;
+            public int lfWeight;
+            public byte lfItalic;
+            public byte lfUnderline;
+            public byte lfStrikeOut;
+            public byte lfCharSet;
+            public byte lfOutPrecision;
+            public byte lfClipPrecision;
+            public byte lfQuality;
+            public byte lfPitchAndFamily;
+            [MarshalAsAttribute(UnmanagedType.ByValTStr, SizeConst = LF_FACESIZE)]
+            public string lfFaceName;
```

Comment:
`lfFaceName` looks like an output will be filled by the `GetThemeFont` function, don't think the function can write to it, StringBuffer or char[] should be used.

Reply:
`UnmanagedType.ByValTStr` [handles this](https://docs.microsoft.com/en-us/dotnet/framework/interop/default-marshaling-for-strings). `lfFaceName` is a fixed-size array of `TCHAR`s (`LF_FACESIZE` = 32).